src/locales/nl/pages.yaml

```
@@ -229,15 +229,15 @@ contact:
 
     ### Bestuur
 
-    **Bestuur 53 "Pompoen"**
+    **Bestuur 54 "Kiwi"**
 
-    Voorzitter: Arthur Schoorlemmer ([voorzitter@krashna.nl](mailto:voorzitter@krashna.nl))\
-    Secretaris: Nienke Brouwer ([secretaris@krashna.nl](mailto:secretaris@krashna.nl))\
-    Penningmeester: Mari&euml;tte Sch&ouml;nfeld ([penningmeester@krashna.nl](mailto:penningmeester@krashna.nl))\
-    Acquisiteur en Kamermuziekcommissaris: Rivka Eikelboom ([acquisiteur@krashna.nl](mailto:acquisiteur@krashna.nl) en [kamucom@krashna.nl](mailto:kamucom@krashna.nl))\
-    Orkestcommissaris: Otto Kaaij ([orkestcom@krashna.nl](mailto:orkestcom@krashna.nl))\
-    Koorcommissaris: Anouk de S&eacute;vaux ([koorcom@krashna.nl](mailto:koorcom@krashna.nl))\
-    Publiciteits- en Concertcommissaris: Jochem Schuitemaker ([publicom@krashna.nl](mailto:publicom@krashna.nl) en [concertcom@krashna.nl](mailto:concertcom@krashna.nl))
+    Voorzitter: Marthe Meskers ([voorzitter@krashna.nl](mailto:voorzitter@krashna.nl))\
+    Secretaris: Pim de Jong ([secretaris@krashna.nl](mailto:secretaris@krashna.nl))\
+    Penningmeester: Duschka Kleijn ([penningmeester@krashna.nl](mailto:penningmeester@krashna.nl))\
+    Acquisiteur: Maurice Hooikamer ([acquisiteur@krashna.nl](mailto:acquisiteur@krashna.nl))\
```

```suggestion
    Acquisiteur: Maurice Hooikammer ([acquisiteur@krashna.nl](mailto:acquisiteur@krashna.nl))\
```
At least, that's his name on Facebook :)

---

Oh oops. you're absolutely right. Thanks :+1: